MeshLib/MeshSurfaceExtraction.cpp

```diff
@@ -124,6 +131,31 @@ MeshLib::Mesh* MeshSurfaceExtraction::getMeshSurface(
             std::copy(id_map.cbegin(), id_map.cend(), orig_node_ids->begin());
         }
     }
+
+    // transmit the original subsurface element ids as a property
+    if (!subsfc_element_id_prop_name.empty()) {
+        boost::optional<MeshLib::PropertyVector<std::size_t>&> orig_elem_ids(
+            result->getProperties().createNewPropertyVector<std::size_t>(
+                subsfc_element_id_prop_name , MeshLib::MeshItemType::Cell, 1));
+        if (orig_elem_ids) {
+            orig_elem_ids->resize(element_ids_map.size());
+            std::copy(element_ids_map.cbegin(), element_ids_map.cend(),
+                      orig_elem_ids->begin());
+        }
+    }
+
+    // transmit the face id of the original subsurface element as a property
+    if (!face_id_prop_name.empty()) {
+        boost::optional<MeshLib::PropertyVector<std::size_t>&> orig_face_ids(
+            result->getProperties().createNewPropertyVector<std::size_t>(
+                face_id_prop_name , MeshLib::MeshItemType::Cell, 1));
+        if (orig_face_ids) {
```

Comment:
I'm not sure if to quit here is the right solution. Maybe a user don't need this information and for this reason gives an empty name string?